frontend/resourceadm/components/PartyListDetails/PartyListDetail.tsx

```diff
@@ -0,0 +1,221 @@
+} from '@digdir/design-system-react';
+import classes from './PartyListDetail.module.css';
+import { PartyList, PartyListMember } from 'app-shared/types/ResourceAdm';
+import { FieldWrapper } from '../FieldWrapper/FieldWrapper';
+import { useEditPartyListMutation } from 'resourceadm/hooks/mutations/useEditPartyListMutation';
+import { useRemovePartyListMemberMutation } from 'resourceadm/hooks/mutations/useRemovePartyListMemberMutation';
+import { useAddPartyListMemberMutation } from 'resourceadm/hooks/mutations/useAddPartyListMemberMutation';
+import { createReplacePatch } from '../../utils/jsonPatchUtils/jsonPatchUtils';
+import { useDeletePartyListMutation } from 'resourceadm/hooks/mutations/useDeletePartyListMutation';
+import { PartyListSearch } from '../PartyListSearch/PartyListSearch';
+
+export interface PartyListDetailProps {
+  org: string;
+  env: string;
+  list: PartyList;
+  backUrl: string;
+}
+
+export const PartyListDetail = ({
+  org,
+  env,
+  list,
+  backUrl,
+}: PartyListDetailProps): React.ReactNode => {
+  const { t } = useTranslation();
+
+  const deleteWarningModalRef = useRef<HTMLDialogElement>(null);
+  const navigate = useNavigate();
+  const [listItems, setListItems] = useState<(PartyListMember & { isDeleted?: boolean })[]>(
+    list.members ?? [],
+  );
+  const [listName, setListName] = useState<string>(list.name || '');
+  const [listDescription, setListDescription] = useState<string>(list.description || '');
+
+  const { mutate: editPartyList } = useEditPartyListMutation(org, list.identifier, env);
+  const { mutate: deletePartyList } = useDeletePartyListMutation(org, list.identifier, env);
+  const { mutate: removeListMember } = useRemovePartyListMemberMutation(org, list.identifier, env);
+  const { mutate: addListMember } = useAddPartyListMemberMutation(org, list.identifier, env);
+
+  // add member
+  const handleAddMember = (memberToAdd: PartyListMember): void => {
+    addListMember(memberToAdd.orgNr);
+    setListItems((old) => [...old, memberToAdd]);
+  };
+
+  // remove member
+  const handleRemoveMember = (memberIdToRemove: string): void => {
+    removeListMember(memberIdToRemove);
+    setListItems((old) =>
+      old.map((x) => (x.orgNr === memberIdToRemove ? { ...x, isDeleted: true } : x)),
+    );
+  };
+
+  // undo remove member
+  const handleUndoRemoveMember = (memberIdToUndoRemove: string): void => {
+    addListMember(memberIdToUndoRemove);
+    setListItems((old) =>
+      old.map((x) => (x.orgNr === memberIdToUndoRemove ? { ...x, isDeleted: false } : x)),
+    );
+  };
+
+  // change list name, description and possibly other properties
+  const handleSave = (diff: Partial<PartyList>): void => {
+    editPartyList(createReplacePatch<Partial<PartyList>>(diff));
+  };
+
+  // slett, må gjøres utenfor? Evt ha en back-funksjon
+  const handleDelete = (): void => {
+    deletePartyList(undefined, {
+      onSuccess: () => navigate(backUrl),
+      onError: (_error: any) => {
+        // TODO
```

Comment:
Is this supposed to be there?
Maybe we can create an issue for the task, and reference it in the code for future work 😄